src/nxdoc/nuxeo-add-ons/nuxeo-ai.md

```diff
@@ -123,6 +123,127 @@ provides 4 default document streams that can be activated by the [configuration
 
 These allow you to start your processing chain quickly.
 
+## Insight Deduplication
+
+_This feature requires an Insight subscription_
+
+The Deduplication feature presents images that are similar to other existing repository images. In a Nuxeo repository, such detection can be run each time a picture will be added to a document. On existing assets, a complete re-index can be operated through Nuxeo Insight.
+
+![]({{file name='deduplication.png'}} ?w=600,border=true)
+
+### Configuration
+
+In order to activate the deduplication feature, we need to activate it in the `nuxeo.conf` file:
+
+```
+nuxeo.insight.dedup.enabled=true
+```
+
+By default, each document of 'Picture' type will be analyzed to detect potential similar documents. Here is the exact query:
+
+```
+SELECT * FROM Document WHERE ecm:primaryType = 'Picture' AND ecm:tag NOT IN ('not_duplicate')
+```
+
+This query and the metadata to introspect can be customized through a Nuxeo extension point as follows:
+
+```
+<requires>org.nuxeo.ai.similar.content.default.config</require>
+<extension target="org.nuxeo.ai.similar.content.services.SimilarServiceComponent" point="configuration">
+    <deduplication name="specific-contribution"
+                   query="SELECT * FROM Document WHERE ecm:primaryType = 'Picture' AND ecm:tag NOT IN ('not_duplicate')">
```

Comment:
Same issue again. use the mixin. 
Customers use the Picture functionality by adding the Picture facet for their Doctypes